### DISCUSSION TITLE
How to handle a lot of pages ?

### DISCUSSION THREAD

**Discussion by jonathanoeijoeng** [gilbitron/laravel-vue-pagination#164] [Help & support]
![image](https://user-images.githubusercontent.com/7046560/229295694-e6510bd8-e811-467d-86f1-b826ae20fe7d.png)

I want to achieve this pagination which I will have 3 dots if I have to many pages.
My script is like below and it show all pages number from 1 -25.
`
<TailwindPagination
        :data="qualities"
        :itemClasses="[
          'bg-slate-50 text-xs hover:bg-colorprimary-100 font-reguler text-slate-400',
        ]"
        :activeClasses="['bg-colorprimary-200 text-xs font-bold']"
        @pagination-change-page="getResults"
      />
`

**Comment by lombervid** [comment]
Specify a `limit` prop with the number of pages you want to be shown on each side of the current page.

https://laravel-vue-pagination.org/guide/api/props.html#limit
